docs/modules/releasenotes/partials/release-notes-24.2.adoc

```diff
@@ -168,3 +168,12 @@ The chart introduces two new properties:
 * `options.plugins.legend.pointsVisible` Whether the colored points in the legend are visible.
 
 For more information about the Chart widget see xref:technical-guide:user-interface/widget-reference.adoc#chart[Chart].
+
+== Slider
+
+The `Slider` is now rendered using multiple `<div>` instead of using the browser native `<input type="range">`.
```

Comment:
```suggestion
The `Slider` widget is now rendered using multiple `<div>` instead of the browser-native `<input type="range">`.
This allows for a more precise adjustment of the appearance.
```

(Not only explain what has changed but also what the benefits are.)